sublime-package.json

```diff
@@ -0,0 +1,417 @@
+                },
+                "tcp_mode": {
+                  "type": "string",
+                  "default": "",
+                  "enum": [
+                    "",
+                    "host",
+                  ],
+                  "markdownDescription": "Set to `\"host\"` if the server connects to the editor. Otherwise, LSP will connect to the server."
+                },
+                "languages": {
+                  "$ref": "#/definitions/ClientLanguages"
+                },
+                "initializationOptions": {
+                  "$ref": "#/definitions/ClientInitializationOptions"
+                },
+                "settings": {
+                  "$ref": "#/definitions/ClientSettings"
+                },
+                "experimental_capabilities": {
+                  "$ref": "#/definitions/ClientExperimentalCapabilities",
+                },
+                "env": {
+                  "$ref": "#/definitions/ClientEnv"
+                },
+                "languageId": {
+                  "type": "string",
+                  "default": "somelang",
+                  "markdownDescription": "The \"Language ID\". It is sent to the language server so that it knows with what kind of file it is dealing with."
+                },
+                "document_selector": {
+                  "type": "string",
+                  "default": "source.somelang",
+                  "markdownDescription": "**ST4 only**. The Sublime Text selector to attach the language server process to applicable views. If not specified, it defaults to \"source.${languageId}\". This is the connection between your files and language servers. It's a selector that is matched against the current view's base scope. If the selector matches with the base scope of the the file, the associated language server is started. If the selector happens to be of the form \"source.{languageId}\" (which it is in many cases), then you can omit this `\"document_selector\"` key altogether, and LSP will assume the selector is \"source.{languageId}\".\n\nFor more information, refer to the [Sublime Text docs on selectors](https://www.sublimetext.com/docs/3/selectors.html)"
+                },
+                "feature_selector": {
+                  "type": "string",
+                  "default": "source.somelang",
+                  "markdownDescription": "**ST4 only**. The Sublime Text selector to use to give preference to this language server when multiple servers are attached to a view. For certain capabilities, like code completion, only one language server is chosen. The selector specified here decides which language server is chosen above others."
+                },
+                "syntaxes": {
+                  "type": "array",
+                  "markdownDescription": "The Sublime Text syntax paths that need to match current file for server to be started."
```

Comment:
Maybe deprecate this and point towards `"languages"` ?

Reply:
Then I would also need to update all `default_clients` and I can't be bothered.